pipeline_logs/firestore_pipeline_logs_table.py

```diff
@@ -0,0 +1,37 @@
+import firebase_admin
+from core_data_modules.logging import Logger
+from firebase_admin import credentials, firestore
+
+
+log = Logger(__name__)
+
```

Comment:
Can you enumerate standard events that have meaning elsewhere? In other words, add
PipelineEvents.PIPELINE_START and another for end.

This is is so we can't mistype the strings for special events that have significance for layout on the dashboard.

Reply:
Thanks added as a dict of pipeline event 
` {"1":"CodaAdd", "2":"FetchingRawData", "3":"GeneratingOutputs", "4":"CodaGet", "5":"GeneratingAutomatedAnalysisFiles", "6":"BackingUpData", "7":"UploadingAnalysisFiles", 8":"UploadingLogFiles", "9":"PipelineCompletedSuccessfully"}`